src/odemis/acq/stream/_sync.py

```diff
@@ -454,13 +470,41 @@ def _getSpotPositions(self):
         logging.debug("Generating points in the SEM area %s, from rep %s and roi %s",
                       lim_main, rep, roi)
 
-        pos = numpy.empty((rep[1], rep[0], 2), dtype=float)
+        pos = numpy.empty((rep[1], rep[0], 2), dtype=float) # in SEM px (at scale = 1)
         posy = pos[:, :, 1].swapaxes(0, 1)  # just a view to have Y as last dim
         posy[:, :] = numpy.linspace(lim_main[1], lim_main[3], rep[1])
         # fill the X dimension
         pos[:, :, 0] = numpy.linspace(lim_main[0], lim_main[2], rep[0])
+
         return pos
 
+    def _getLeftTopPositionPhys(self) -> Tuple[float, float]:
+        """
+        Compute the position of the top-left pixel of the RoA in physical coordinates (m).
+        Note that this is *not* the top-left corner of the image. It is shifted by half a pixel.
+        :return: x, y position of the center of pixel (m,m) in absolute coordinates
```

Comment:
I don't fully understand what is returned. Is it the top left position in physical coordinates, measured with the center of the image as origin?

Reply:
No, the position (x, y) of the top-left pixel center of the RoA in absolute coordinates. "Absolute" means it's in the stage coordinates. I've updated the docstring.